### ISSUE TITLE
detail about USE

### ISSUE THREAD

**Issue by Cumberbatch08** [tensorflow/hub#433]
first, many thanks to the code, I can get a fixed length vector for a text. But I can't get much more details about the USE model such as the layers, dataset, loss function. Can you support the train process?
Because I want to compare USE with SBERT.

**Comment by rmothukuru** [comment]
@Cumberbatch08,
Can you please refer to this [Research Paper](http://ceur-ws.org/Vol-2431/paper2.pdf) and let us know if it helps. Thanks!

**Comment by Cumberbatch08** [comment]
@rmothukuru , the paper link[](https://arxiv.org/pdf/1803.11175.pdf)

**Comment by gowthamkpr** [comment]
Closing the issue as it has been inactive for more than a month. Please add additional comments and we can open the issue again. Thanks!

**Comment by maximneumann** [comment]
There are multiple USE hub modules on tfhub:
https://tfhub.dev/s?q=universal-sentence-encoder

If you want to learn more, I'd recommend to try out the colabs and the examples on the individual module pages.
To learn more about the details, please check the publications and refer to code implementations (currently hub modules don't provide information about the code).
